projects/packages/sync/src/modules/class-themes.php

```diff
@@ -77,7 +75,7 @@ public function sync_widget_edit( $instance, $new_instance, $old_instance, $widg
 		}
 
 		// Don't trigger sync action if this is an ajax request, because Customizer makes them during preview before saving changes.
-		// phpcs:disable WordPress.Security.NonceVerification.Missing
+		// phpcs:disable WordPress.Security.NonceVerification.Missing -- We are using $_POST['customized'] to early return if this is an ajax request from Customizer.
```

Comment:
I might make this slightly more explicit:
```suggestion
		// phpcs:disable WordPress.Security.NonceVerification.Missing -- Not doing anything with or in response to the $_POST data. We're only using $_POST['customized'] to early return if this is an ajax request from Customizer.
```